src/bp-moderation/bp-moderation-functions.php

```diff
@@ -199,6 +199,22 @@ function bp_moderation_get_content_excerpt( $moderation_item_id, $moderation_ite
 	return $content_excerpt;
 }
 
+/**
+ * Function to get specific moderation content type.
+ *
+ * @since BuddyBoss 1.5.4
+ *
+ * @param string $key content type key.
+ *
+ * @return mixed|void
+ */
+function bp_get_moderation_content_type( $key ) {
```

Comment:
@janakkaneriya007  this function not needed we have `bp_moderation_get_content_type`